product/src/main/java/com/yas/product/viewmodel/product/ProductPutVm.java

```diff
@@ -38,6 +38,7 @@ public record ProductPutVm(
         List<Long> productImageIds,
         List<ProductVariationPutVm> variations,
         List<ProductOptionValuePutVm> productOptionValues,
+        List<ProductOptionValueDisplay> productOptionValueDisplay,
```

Comment:
Since `productOptionValueDisplay` is a list, could we rename it to `productOptionValueDisplays`? This would make it more evident that it holds multiple items.